tests/rptest/services/redpanda.py

```diff
@@ -1509,7 +1510,10 @@ def set_cluster_config(self,
         def is_ready():
             status = admin_client.get_cluster_config_status(
                 node=self.controller())
-            ready = all([n['config_version'] >= new_version for n in status])
+            ready = all([
```

Comment:
Let make it explicit what it does; right now its meaning a bit hidden:

```
if skip_readiness_check:
  config_status = admin_client.get_cluster_config_status(node=self.controller())
else:
  config_status = wait_until_result(
    is_ready,
    ...
  )
```

Reply:
np, we should use `any` to check that the change is applied at least on a single node

Reply:
i dropped this part as it is not longer required